cwltool/process.py

```diff
@@ -616,9 +617,10 @@ def _init_job(self, joborder, runtimeContext):
             tmpdir = runtimeContext.docker_tmpdir or "/tmp"
             stagedir = runtimeContext.docker_stagedir or "/var/lib/cwl"
         else:
-            outdir = fs_access.realpath(runtimeContext.outdir or
-                tempfile.mkdtemp(prefix=getdefault(runtimeContext.tmp_outdir_prefix,
-                                                   DEFAULT_TMP_PREFIX)))
+            outdir = (runtimeContext.outdir or
+                      tempfile.mkdtemp(prefix=getdefault(runtimeContext.tmp_outdir_prefix,
+                                                         DEFAULT_TMP_PREFIX)))
+            outdir = fs_access.realpath(outdir)
```

Comment:
Why? I wouldn't call `fs_access.realpath` on the output of `tempfile.mkdtemp`..

Reply:
I did not mean to change the behaviour here, just the formatting. having said that, I understood that `fs_access.realpath` was already being called on the output of `tempfile.mkdtemp` Did I misunderstand the statement?

Reply:
Sorry, missed that this was a pure reformat. I'll investigate further, elsewhere.